tree/dataframe/src/RLoopManager.cxx

```diff
@@ -504,7 +504,7 @@ void RLoopManager::RunTreeReader()
 void RLoopManager::RunDataSource()
 {
    assert(fDataSource != nullptr);
-   fDataSource->Initialise();
```

Comment:
You probably need to call both versions here, for backward compatibility? You can even test that by moving only half the data sources to the new spelling and checking whether both new and old still work.